versioned_docs/version-0.4.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx

```diff
@@ -117,7 +117,8 @@ spec:
 
 :::caution
 The `kubeslice-cli create sliceConfig` command returns after the configuration is applied. However, in each cluster, the relevant pods for 
```

Comment:
What does the command return?

Reply:
This was the note given by Eric to add to the doc.

Reply:
The command returns after the configuration is applied - the commands takes time to return

Reply:
The command returns the response/expected output. 

We should write about what the command returns...

Reply:
rephrased it.